pyproject.toml

```diff
@@ -25,20 +25,19 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.8,<3.11"
-geoh5py = "~0.8.0a1"
-# geoh5py = {url = "https://github.com/MiraGeoscience/geoh5py/archive/refs/heads/GEOPY-715.zip#sha256=None"}
-numpy = "!=1.19.4, ~1.23.5"
-
+#geoh5py = {version = "~0.8.0a1", allow-prereleases = true}
+#geoh5py = {url = "https://github.com/MiraGeoscience/geoh5py/archive/refs/heads/develop.zip#sha256=None"}
+#geoh5py = {path = "../geoh5py", develop = true}
+geoh5py = {url = "https://github.com/sebhmg/geoh5py/archive/refs/heads/GEOPY-947.zip#sha256=None"}
```

Comment:
**chore:** before merging, will revert to using head/develop after changes are committed in geoh5py